launch/interaction/vizbox.launch

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+
+<launch>
+
+    <!-- Get the machine file -->
+    <arg name="machine" default="localhost"/>
```

Comment:
Do we still use machine files?

Reply:
It is all the files, so keep it that way for when we want to use it again.